lib/livebook/notebook/export/elixir.ex

```diff
@@ -0,0 +1,80 @@
+defmodule Livebook.Notebook.Export.Elixir do
```

Comment:
In the future we would most likely make this part of the language but not a concern until we decide to support multiple languages. :D

Reply:
It would probably be sufficient to know comment prefix for the given language, but yeah.